easybuild/easyconfigs/r/R/R-3.6.2-foss-2019b.eb

```diff
@@ -243,7 +249,11 @@ exts_list = [
         'checksums': ['faa25754b757fe483b876f5d07b73f76f69a1baa971420892fadec4af4bbad21'],
     }),
     ('cubature', '2.0.4', {
-        'checksums': ['d97ce5eaac5e43910208e8274ddf6ff4f974d05688f0247ebccd807e24c2fe4a'],
+        'patches': ['cubature_2.0.4_config_guess.patch'],
+        'checksums': [
+            'd97ce5eaac5e43910208e8274ddf6ff4f974d05688f0247ebccd807e24c2fe4a',  # cubature_2.0.4.tar.gz
+            '6bff93402edd6e9e81d9af8ea471d1bf2baeb817bde21faeade4575eeca54227',  # cubature_2.0.4_config_guess.patch
```

Comment:
@edmondac Can we avoid patching the `config.guess`, since this is a pretty massive patch...

In `ConfigureMake` we have some logic where we download & cache a recent `config.guess`, maybe that could be leveraged in `RPackage` (on demand)?

Reply:
We'll have the discussion in #9639 then follow that decision here :-)